### ISSUE TITLE
Docs incorrect for .NET Core?

### ISSUE THREAD

**Issue by nb77qlik** [dotnet/dotnet-api-docs#1269]
I don't think these docs are correct for .NET Core 2.1 - see https://github.com/dotnet/corefx/issues/23422
Behaviour seems to be different between .NET Framework & .NET Core.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c67352df-ccc0-0cc7-51e7-998f9714580f
* Version Independent ID: 4e029cef-0c24-1f85-2e8d-c96eb4bc71ed
* Content: [HttpWebRequest.AllowAutoRedirect Property (System.Net)](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.allowautoredirect?view=netcore-2.1)
* Content Source: [xml/System.Net/HttpWebRequest.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net/HttpWebRequest.xml)
* Product: **dotnet-api**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

**Comment by karelz** [comment]
That is correct. The behavior on .NET Core is different for `ServicePoint(Manager)` and for `HttpWebRequest`. We need to document that.
We could use some help if someone is interested.

**Comment by lethienhoang** [comment]
Hi. I have been working as c# developer since 2016.  I want to contribute open sources but I don;t have experience much. Who can help me?

**Comment by nemrism** [comment]
Hi there,

Do you still expect some help on documenting the different behavior between .NET framework and .net core?

After digging into the issues in [dotnet/corefx](https://github.com/dotnet/corefx/issues) repository, I understand the missing information as ".net core doesn't allow automatic redirect from an https:// request to a http:// location for security reasons".

@karelz, is that what you meant by "We need to document that"?
